src/server/controllers/email-subscriptions-controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Request, Response } from 'express';
+import * as mailchimp from 'mailchimp-api';
+
+const MC = new mailchimp.Mailchimp(process.env.MAILCHIMP_API_KEY);
+
+export function createSubscription(req: Request, res: Response) {
+  // TODO: Check user is already subscribed
+  MC.lists.subscribe({
+    id: process.env.MAILCHIMP_INTERESTED_LIST_ID,
+    email: { email: req.body.email },
+    merge_vars: { EMAIL: req.body.email },
+    update_existing: true
+  },
+  _ => res.json('An email requesting your confirmation has been sent to the address you provided.'),
+  err => err.name === 'ValidationError' ? res.status(400).json(err.error) : res.status(500).json(err.error));
```

Comment:
In general I think we should avoid displaying unknown error messages in the response, in case we end up leaking information.  Rather we should display a generic error message to the user and log the actual error on the server.

Reply:
The validation errors provided by MailChimp are quite useful to end-users since they're very specific, but we could still send a generic error for other errors, and also add server-side validation so that we avoid relying on MailChimp for validation errors

Reply:
If we can trust that the validation errors won't leak info, then that sounds to me like a good approach 👍